Source/santad/SNTExecutionController.m

```diff
@@ -191,22 +193,24 @@ - (void)validateBinaryWithMessage:(santa_message_t)message {
       }
 
       if (!cd.silentBlock) {
-        // Let the user know what happened, both on the terminal and in the GUI.
-        NSAttributedString *s = [SNTBlockMessage attributedBlockMessageForEvent:se
-                                                                  customMessage:cd.customMsg];
-        NSMutableString *msg = [NSMutableString stringWithCapacity:1024];
-        [msg appendFormat:@"\n\033[1mSanta\033[0m\n\n%@\n\n", s.string];
-        [msg appendFormat:@"\033[1mPath:      \033[0m %@\n"
-                          @"\033[1mIdentifier:\033[0m %@\n"
-                          @"\033[1mParent:    \033[0m %@ (%@)\n\n",
-            se.filePath, se.fileSHA256, se.parentName, se.ppid];
-        NSURL *detailURL = [SNTBlockMessage eventDetailURLForEvent:se];
-        if (detailURL) {
-          [msg appendFormat:@"More info:\n%@\n\n", detailURL.absoluteString];
-        }
-        [self printMessage:msg toTTYForPID:message.ppid];
-
-        [_notifierQueue addEvent:se customMessage:cd.customMsg];
+        dispatch_async(_eventQueue, ^{
```

Comment:
As discussed, given that the property should always have been set before reaching this code, neither should need to be on the queue. Unless I'm missing something.